.cargo/config.toml

```diff
@@ -2,4 +2,4 @@
 protocol = "sparse"
 
 [build]
-target = ["x86_64-unknown-linux-gnu", "i686-unknown-linux-gnu", "aarch64-linux-android", "armv7-linux-androideabi", "i686-linux-android", "x86_64-linux-android"]
```

Comment:
The build failed on some platforms because the `prctl` crate does not support Android. I am not sure how useful the setup feature would be on Android anyway, since phones are usually unrooted and you don't have permissions for this kind of network setup. (Would this be useful on a rooted Android?) However, I have turned off that feature on non-Linux platforms for now in cb1babebd4db4b5fc0ba4a0218368f056eff0847, so maybe you get to build it for Android with that change. (I get a linker error at the moment, but at least it compiles now.)

If it would be useful on Android, the long-term goal would consist in adding Android support to the `prctl` crate or adding `prctl` to the `nix` crate, along with Android support.